data/test/scenarios/test_movetype.cfg

```diff
@@ -332,29 +332,164 @@
     [/modify_unit]
 #enddef
 
+#####
+# API(s) being tested: N/A
```

Comment:
All of the `MOVETYPE_MOVE_SCEN` tests in this file are also testing `[find_path]`.